config/nginx-default.conf

```diff
@@ -0,0 +1,12 @@
+server {
+    server_name profiles--dev;
+    listen 80;
+
+    location / {
+        uwsgi_pass wsgi:9000;
```

Comment:
The `nginx` and `profiles_wsgi` containers communicate over a port, so they do not depend on a shared filesystem.